src/components/browse/components/FacetList/FacetTermsList.js

```diff
@@ -405,7 +481,7 @@ export const CountIndicator = React.memo(function CountIndicator({ count = 1, co
     });
     return (
         <svg className="svg-count-indicator" viewBox={`0 0 ${width + 2} ${height + 2}`} width={width + 2} height={height + 2}>
-            { dots }
+            { dots}
```

Comment:
Haha
I guess one idea for us might be to setup/define code linting config file more, though on other hand it can be glitchy too or overly verbose/strict.